nitro-protocol/src/contract/multi-asset-holder.ts

```diff
@@ -64,126 +64,65 @@ export function convertAddressToBytes32(address: string): string {
  * @param allocation
  * @param indices
  */
-export function computeClaimEffectsAndInteractions(
-  initialHoldings: string,
+export function computeReclaimEffects(
   sourceAllocations: ExitFormat.Allocation[], // we must index this with a JS number that is less than 2**32 - 1
   targetAllocations: ExitFormat.Allocation[], // we must index this with a JS number that is less than 2**32 - 1
-  indexOfTargetInSource: number,
-  targetAllocationIndicesToPayout: number[]
-): {
-  newSourceAllocations: ExitFormat.Allocation[];
-  newTargetAllocations: ExitFormat.Allocation[];
-  exitAllocations: ExitFormat.Allocation[];
-  totalPayouts: string;
-} {
-  let totalPayouts = BigNumber.from(0);
-  let k = 0;
+  indexOfTargetInSource: number
+): ExitFormat.Allocation[] {
+  const newSourceAllocations: ExitFormat.Allocation[] = []; // will be one slot shorter than sourceAllocations
```

Comment:
It is a useful comment. I have copied it to `MultiAssetHolder.compute_reclaim_effects`.